src/DataViews/LiveFunctionsDataView.cpp

```diff
@@ -142,41 +143,37 @@ void LiveFunctionsDataView::UpdateSelectedFunctionId() {
   selected_function_id_ = app_->GetHighlightedFunctionId();
 }
 
-std::vector<uint64_t> LiveFunctionsDataView::GetFunctionTimerDurations(uint64_t function_id) {
-  ORBIT_SCOPE_FUNCTION;
-  const std::vector<const orbit_client_protos::TimerInfo*> timers =
-      app_->GetAllTimersForHookedFunction(function_id);
-  std::vector<uint64_t> timer_durations;
-  std::transform(std::begin(timers), std::end(timers), std::back_inserter(timer_durations),
-                 [](const orbit_client_protos::TimerInfo* const timer) {
-                   return timer->end() - timer->start();
-                 });
-
-  std::sort(timer_durations.begin(), timer_durations.end());
-  return timer_durations;
-}
-
 void LiveFunctionsDataView::UpdateHistogramWithIndices(
     const std::vector<int>& visible_selected_indices) {
   std::vector<uint64_t> function_ids;
   std::transform(std::begin(visible_selected_indices), std::end(visible_selected_indices),
                  std::back_inserter(function_ids),
                  [this](const int index) { return indices_[index]; });
 
-  UpdateHistogramWithFunctionIds(function_ids);
+  UpdateHistogramWithScopeIds(function_ids);
 }
 
-void LiveFunctionsDataView::UpdateHistogramWithFunctionIds(
-    const std::vector<uint64_t>& function_ids) {
-  if (function_ids.empty() || !timer_durations_.contains(function_ids[0])) {
-    app_->ShowHistogram(nullptr, "", orbit_grpc_protos::kInvalidFunctionId);
+void LiveFunctionsDataView::UpdateHistogramWithScopeIds(const std::vector<uint64_t>& scope_ids) {
+  const std::vector<uint64_t>* timer_durations =
+      app_->HasCaptureData() && !scope_ids.empty()
```

Comment:
```suggestion
      (app_->HasCaptureData() && !scope_ids.empty())
```
Quite unreadable otherwise.